exploration/number-selection.md

```diff
@@ -92,44 +95,360 @@ ICU MF1 messages using `plural` and `selectordinal` should be representable in M
+> one  {{{$numDays} den}}
+> few  {{{$numDays} dny}}
+> many {{{$numDays} dne}}
+> *    {{{$numDays} dní}}
+> ```
+> Using the rules found above, the results of various `operand` values might look like:
+> | Operand value | Keyword | Formatted Message |
+> |---|---|---|
+> | 1 | `one` | 1 den |
+> | 2 | `few` | 2 dny |
+> | 5 | `other` | 5 dní |
+> | 22 | `few` | 22 dny |
+> | 27 | `other` | 27 dní |
+> | 2.4 | `many` | 2,4 dne |
+
+
+
+### Determining Exact Literal Match
+
+> [!IMPORTANT]
+> The exact behavior of exact literal match is only defined for non-zero-filled
+> integer values.
+> Annotations that use fraction digits or significant digits might work in specific
+> implementation-defined ways.
+> Users should avoid depending on these types of keys in message selection.
+
+
+Number literals in the MessageFormat 2 syntax use the 
+[format defined for a JSON number](https://www.rfc-editor.org/rfc/rfc8259#section-6).
+The resolved value of an `operand` exactly matches a numeric literal `key`
+if, when the `operand` is serialized using the format for a JSON number
+the two strings are equal.
+
+> [!NOTE]
+> Implementations are not expected to implement this exactly as written,
+> as there are clearly optimizations that can be applied.
+
+> [!NOTE]
+> Only integer matching is required in the Technical Preview.
+> Feedback describing use cases for fractional and significant digits-based
+> selection would be helpful.
+> Otherwise, users should avoid using matching such as the following examples.
+
+> **Examples.**
+> ```
+> .local $num = {|1.0| :number}
+> .match {$num :number}
+> 1     {{Matches}}
+> 1.0   {{Does not match}}
+> 1.00  {{Does not match}}
+>
+> .local $num = {|1.0| :number}
+> .match {$num :number minimumFractionDigits=1}
+> 1     {{Does not match}
+> 1.0   {{Matches}}
+> 1.00  {{Does not match}}
+>
+> .local $num = {|1.0| :number}
+> .match {$num :number minimumFractionDigits=2}
+> 1     {{Does not match}
+> 1.0   {{Does not match}}
+> 1.00  {{Matches}}
+> 
+> .local $num = {|-1.00001| :number}
+> .match {$num :integer}
+> -1     {{Matches}}
+> -1.0   {{Does not match}}
+> -1.00  {{Does not match}}
+>
+> .local $num = {|-1.00001| :number}
+> .match {$num :number minimumFractionDigits=1}
+> -1        {{Does not match}
+> -1.0      {{Matches}}
+> -1.00001  {{Does not match}}
+> 
+> .local $num = {|-1.00001| :number}
+> .match {$num :number}
+> -1        {{Does not match}
+> -1.0      {{Does not match}}
+> -1.00001  {{Matches}}
+>
+> .local $num = {|-1.00001|}
+> .match {$num :number maximumSignificantDigits=1}
+> -1        {{Matches}}
+> -1.0      {{Does not match}}
+> -1.00001  {{Does not match}}
+>
+> .local $num = {1.3}
+> .match {$num :number minimumSignificantDigits=3}
+> 1    {{Does not match}}
+> 1.3  {{Does not match}}
+> 1.30 {{Matches}}
+>
+> .local $num = {1234}
+> .match {$num :number maximumSignificantDigits=2}
+> 1200   {{Matches}}
+> 1234   {{Does not match}}
+> 1200.0 {{Does not match}}
+> ```
```

Comment:
Including these examples seems distracting, as they're showing messages that we're saying _should not_ be used, and they're indicating that specific variants would match.

```suggestion
```

Reply:
I agree but want to keep them for this morning's discussion **_only_**